glean/schema/source/src.angle

```diff
@@ -80,7 +80,8 @@ type ByteSpans = [RelByteSpan]
 # Kind of language supported by Glean
 type Language = enum {
   Buck | C | Cpp | Hack | Haskell  |
-  ObjC | ObjCpp | Python | Thrift | Java | GraphQL
+  ObjC | ObjCpp | Python | Thrift | Java | GraphQL |
+  JavaScript | Flow | TypeScript
```

Comment:
Hmm, I guess this would require a new version of the `src` schema. But I'm surprised the ` src` schema is still in version 1.
Have we made overriting in-place changes like this to the `src` schema before?